creator-node/src/components/README.md

```diff
@@ -1,3 +1,5 @@
+**Note - this pattern has been deprecated. New pattern defined in `creator-node/src/README.md`.**
```

Comment:
can we just remove this file if this README is no longer applicable?

Reply:
its still applicable to all the modules currently in `components`. i left it bc its a useful reference until all the code is migrated away from this structure